README.md

```diff
@@ -39,4 +39,4 @@ Now add yours!
 
 ---
 
-
+### Cb34r was here
```

Comment:
Hey Collin! Great job with this code here! 

Our of curiosity, what does the "#" sign do?

Reply:
Hi Mikel,

the # sign takes a line of txt and increases the font size and weight

Regards,
Collin